android-pay/src/main/java/com/stripe/wrap/pay/AndroidPayConfiguration.java

```diff
@@ -103,6 +110,37 @@ public static AndroidPayConfiguration init(
         mUsesSources = shouldUseSources;
     }
 
+    /**
+     * Specifies a set of card networks to allow. If not explicitly set,
+     * the default supported networks will be AMEX, DISCOVER, MASTERCARD, and VISA.
+     *
+     * @param networks one or more {@link WalletConstants.CardNetwork} values
+     */
+    public void addAllowedCardNetworks(@WalletConstants.CardNetwork Integer... networks) {
```

Comment:
This might be worth adding in a later PR; this isn't really a commonly used feature (to the point where Google demos don't even show it, which is how I missed it in the first place).